xbmc/video/VideoDatabase.cpp

```diff
@@ -1386,6 +1407,48 @@ int CVideoDatabase::AddToTable(const std::string& table, const std::string& firs
   return -1;
 }
 
+int CVideoDatabase::AddRatings(int mediaId, const char *mediaType, const RatingMap& values, const std::string& default)
```

Comment:
`default` is a reserved keyword: http://en.cppreference.com/w/cpp/keyword/default